cfstore/plugins/posix.py

```diff
@@ -1,4 +1,6 @@
 import os
+import cf
+import json
 from cfstore import db
 from cfstore.plugins.ssh import SSHlite
 from cfstore.cfparse_file import cfparse_file
```

Comment:
I suspect I am responsible for this, but this would be a good time to wonder whether the separation of concerns between posix (for file handling) and parsing files for variable metadata is quite right here?